File: mclimate/src/main/java/com/mordrum/mclimate/common/biome/BiomeFrozenLake.java
package com.mordrum.mclimate.common.biome;

import com.mordrum.mclimate.MClimate;
import net.minecraft.util.ResourceLocation;
import net.minecraft.world.biome.Biome;
import net.minecraft.world.biome.BiomeRiver;

public class BiomeFrozenLake extends BiomeRiver {
	public BiomeFrozenLake() {
		super(new Biome.BiomeProperties("FrozenLake").setBaseHeight(-0.5F).setHeightVariation(0.0F).setTemperature(0.0F).setRainfall(0.5F).setSnowEnabled());
		this.spawnableCreatureList.clear();
		this.setRegistryName(new ResourceLocation(MClimate.MOD_ID, "frozen_lake"));
	}
}

File: mclimate/src/main/java/com/mordrum/mclimate/common/biome/BiomeFrozenLake.kt
package com.mordrum.mclimate.common.biome

import com.mordrum.mclimate.MClimate
import net.minecraft.util.ResourceLocation
import net.minecraft.world.biome.Biome
import net.minecraft.world.biome.BiomeRiver

class BiomeFrozenLake : BiomeRiver(Biome.BiomeProperties("FrozenLake").setBaseHeight(-0.5f).setHeightVariation(0.0f).setTemperature(0.0f).setRainfall(0.5f).setSnowEnabled()) {
    init {
        this.spawnableCreatureList.clear()
        this.registryName = ResourceLocation(MClimate.MOD_ID, "frozen_lake")
    }
}
